cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -162,6 +239,56 @@ def listen_for_course_catalog_info_changed(sender, signal, **kwargs):
     )
 
 
+@receiver(COURSE_CERTIFICATE_CONFIG_CHANGED)
+def listen_for_course_certificate_config_changed(sender, signal, **kwargs):
+    """
+    Send course certificate config data onto the Credentials service to create or change one.
+    """
+    certificate_config = kwargs['certificate_config']
+    files_to_upload = dict(get_certificate_signature_assets(certificate_config))
+    try:
+        credentials_client = get_credentials_api_client(
+            User.objects.get(username=settings.CREDENTIALS_SERVICE_USERNAME),
+        )
+        credentials_api_base_url = get_credentials_api_base_url()
+        api_url = urljoin(f'{credentials_api_base_url}/', 'course_certificates/')
+        payload = attr.asdict(certificate_config)
+        payload['course_key'] = str(payload.pop('course_key'))
```

Comment:
Change it to `course_id` pls. Because the API takes the course_id argument by default.

Reply:
Agree and I've fixed it already here https://github.com/raccoongang/edx-platform/pull/2465/files. Please look it also. This PR is going to be merged together with the current one.

Reply:
It's ok that you pass `course_key` to CertificateConfigData here, but in RP https://github.com/raccoongang/edx-platform/pull/2465/files you pass `course_id`?

Reply:
Do not understand the question a little bit. There is no already `course_key` at all. Only we get now `course_id`. I've changed it everwhere. Please explain what do u mean.)